read/app/src/main/java/com/google/samples/quickstart/engagesdksamples/read/converters/ResourceIdToImage.java

```diff
@@ -29,6 +30,7 @@ public static Image convert(int imageResourceId) {
         .setImageUri(getImageUriFromResourceId(imageResourceId))
         .setImageHeightInPixel(IMAGE_HEIGHT)
         .setImageWidthInPixel(IMAGE_WIDTH)
+        .setImageTheme(ImageTheme.IMAGE_THEME_UNSPECIFIED)
```

Comment:
Since the default value of the theme is "UNSPECIFIED", it will be hard to tell if this is picking the user provided value or the default value. Let's use Light or Dark theme

Reply:
Changed to IMAGE_THEME_LIGHT